CoreWhatsNewView/src/au/gov/asd/tac/constellation/views/whatsnew/whatsnew.txt

```diff
@@ -1,6 +1,10 @@
 == 3030-12-31 Getting Started
 <p>If you're new to Constellation, read the <a href="" helpId="au.gov.asd.tac.constellation.functionality.gettingstarted">getting started guide</a>.</p>
 
+== 2025-11-11 Add New Experimental Histogram View
+<p>New Experimental Histogram View added. This is a complete rewrite that can take advantage of JavaFX features present in other views
```

Comment:
Can we also make this more user-friendly? Something like "A new version of the histogram view has been added to the Experimental menu. This aligns the look and feel to be closer to that of other views." (basically remove the mention of JavaFX if possible).